src/Features/Core/Portable/CodeRefactorings/SyncNamespace/AbstractChangeNamespaceService.cs

```diff
@@ -552,7 +554,8 @@ private static async Task<ImmutableArray<ReferencedSymbol>> FindReferencesAsync(
             Debug.Assert(containersToAddImports.Length > 0);
 
             // Need to import all containing namespaces of old namespace and add them to the document (if it's not global namespace)
-            var namesToImport = CreateAllContainingNamespaces(oldNamespace);
+            // as well as the new namespace in case it's needed
```

Comment:
I'd like to explicitly call out why the new namespace might be needed :) 
i.e. there might be multiple namespace decls in the document, and they can contain references to members of the affected namespace.